about.html

```diff
@@ -0,0 +1,202 @@
+                <span class="bi bi-chevron-right ms-2 mt-2"></span>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="about-logo">
+        <div class="container">
+          <h4 class="text-center mt-5">CoWorking Space New 2022 Logo</h4>
+          <div class="small-border mx-auto my-3"></div>
+          <p class="px-sm-5 mx-sm-5 text-center text-gray">
+            Lorem ipsum dolor sit, amet consectetur adipisicing elit. Nostrum iste nulla cum ipsam, illo eveniet
+            consequatur, laudantium minus dolores sapiente accusamus officiis nobis voluptate provident aut repellat
+            maiores totam quibusdam.
+          </p>
+          <div class="d-flex justify-content-center py-5">
+            <img src="./images/logo.png" class="about-logo-img" alt="logo" />
+          </div>
+        </div>
+      </section>
+      <hr />
+      <section class="about">
+        <div class="container">
+          <h4 class="text-center fw-bold mt-5">See The Past Startups From CoWorking Space</h4>
+          <div class="small-border mx-auto my-3"></div>
+          <p class="text-center text-gray">
+            Take a look at the last two years of expreinces from startups at CoWorking Space environment.
+          </p>
+          <div class="d-flex flex-wrap gap-4 justify-content-center py-5">
+            <img src="./images/about_2013.png" class="col-12 col-sm-5" alt="" />
+            <img src="./images/about_2011.png" class="col-12 col-sm-5" alt="" />
+          </div>
+        </div>
+      </section>
+      <section class="partners py-5 bg-gray d-sm-none">
+        <div class="container">
+          <div class="text-center">
+            <h2 class="h3 mt-4 text-light-gray">Partners</h2>
+            <div class="small-border mx-auto"></div>
+          </div>
+          <div class="partners-logo d-flex flex-wrap gap-4 justify-content-around my-5">
+            <img src="./images/amazon.png" alt="" />
+            <img src="./images/google.png" alt="" />
+            <img src="./images/paypal.png" alt="" />
+            <img src="./images/netflix.png" alt="" />
+            <img src="./images/airbnb.png" alt="" />
+          </div>
+        </div>
+      </section>
+    </main>
+    <footer class="container py-5 d-sm-none">
+      <div class="row align-items-center my-4">
+        <div class="col-12 col-md-3 mb-4 mb-sm-0">
+          <img src="./images/logo.png" class="w-100" alt="" />
+        </div>
+        <div class="col-12 col-md-9 px-5 text-gray">
+          <h6 class="fw-bold">2022 CoWorking Space Afghanistan, Some Rights Reserved.</h6>
+          <p>
+            All content on the website, unless otherwise specified, is available under a Creative Commons Attribution
+            2.0 Afghanistan license.
+          </p>
+        </div>
+      </div>
+    </footer>
```

Comment:
- [ ] Please make sure that the footer matches Behance design in the mobile version the logo should be aligned to the right and the text to the left
 
| Your design | Behance |
| ------------- | --------- |
| ![image](https://user-images.githubusercontent.com/74744787/163283702-c3ba7c58-621f-46c9-ba0b-36cc9357abd1.png) | ![image](https://user-images.githubusercontent.com/74744787/163283596-7cf3e25a-ad21-45a3-833d-bd4869802b18.png) |